noir-projects/Earthfile

```diff
@@ -41,10 +41,25 @@ build:
   SAVE ARTIFACT noir-protocol-circuits
 
 test:
-  FROM +build
-  RUN cd noir-protocol-circuits && nargo test --silence-warnings
-  RUN cd aztec-nr && nargo test --silence-warnings
-  RUN cd noir-contracts && nargo test --silence-warnings
+  FROM ../yarn-project/+txe
+
+  # Install nargo
+  COPY ../noir/+nargo/nargo /usr/bin/nargo
```

Comment:
It wasn't needed because the target was `+build` from this package, which includes nargo. The new `FROM` directive is from a TS project that doesn't include it. It was the only way I could find that earthly would behave and let me run TXE without weird dependency errors.